### PR TITLE
adding docs link ISSUE  #972

### LINKED ISSUE #972: add docusaurus.io documentation link in readme.md file and what is docusaurus.io ?
link https://docusaurus.io/docs

### PR DESCRIPTION
**Pull request by manzil-infinity180**
## Overview

Resolves #972 

## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

**Comment by manzil-infinity180** [comment]
Issue https://github.com/celestiaorg/docs/issues/972

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Celestia Docs Page
 
 Celestia Docs is built using [Docusaurus 2](https://docusaurus.io), a modern static website generator.
+# What is Docusaurus
```
Comment (jcstein) [inline README.md:4]:
```suggestion

```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # Celestia Docs Page
 
 Celestia Docs is built using [Docusaurus 2](https://docusaurus.io), a modern static website generator.
+# What is Docusaurus 
+Docusaurus is a static-site generator. It builds a single-page application with fast client-side navigation, leveraging the full power of React to make your site interactive. It provides out-of-the-box documentation features but can be used to create any kind of site (personal website, product, blog, marketing landing pages, etc).
```
Comment (jcstein) [inline README.md:5]:
```suggestion
Learn more in the
```

File: README.md [hunk 3]
```diff
@@ -1,6 +1,10 @@
...
+
```
Comment (jcstein) [inline README.md:6]:
```suggestion

```

File: README.md [hunk 4]
```diff
@@ -1,6 +1,10 @@
...
+[Docusaurus Documentation ](https://docusaurus.io/docs)
```
Comment (jcstein) [inline README.md:7]:
```suggestion
[Docusaurus Documentation](https://docusaurus.io/docs)
```

**Comment by manzil-infinity180** [comment]
Hello sir
What is the procedure for merge of any #PR as I am beginner to open source
I am not know much about this ???

After the raising PR is any thing I have to done with this???

**Comment by jcstein** [comment]
hi yes please just remove the extra line (5) between 4 and 6

**Review by jcstein — APPROVED** [review]

**Pull request merged**